src/tasks/component-installers/dex.ts

```diff
@@ -0,0 +1,326 @@
+                  certs.add([{
+                    title: 'Use default k8s certificate',
+                    task: async (_ctx: any, task: any) => {
+                      task.title = `${task.title}...[OK]`
+                    },
+                  }])
+                  return certs
+                }
+
+                if (!await this.kube.getSecret(this.tlsSecretName, this.namespaceName)) {
+                  const certManager = new CertManagerTasks(this.flags)
+                  certs.add(certManager.getDeployCertManagerTasks(this.flags))
+                  certs.add(certManager.getGenerateCertManagerCACertificateTasks(this.flags))
+                  certs.add(certManager.getCreateCertificateIssuerTasks(this.flags))
+
+                  const domain = 'dex.' + this.flags.domain
+                  const commonName = '*.' + domain
+                  const dnsNames = [domain, commonName]
+                  certs.add(certManager.getGenerateCertificatesTasks(this.flags, commonName, dnsNames, this.tlsSecretName, this.namespaceName))
+                }
+
+                certs.add([{
+                  title: 'Read Dex certificate',
+                  task: async (ctx: any, task: any) => {
+                    const secret = await this.kube.getSecret(this.tlsSecretName, this.namespaceName)
+                    if (secret && secret.data) {
+                      ctx[DexContextKeys.DEX_CA_CRT] = base64Decode(secret.data['ca.crt'])
+                      task.title = `${task.title}...[OK]`
+                    } else {
+                      throw new Error(`Dex certificate not found in the secret '${this.tlsSecretName}' in the namespace '${this.namespaceName}'.`)
+                    }
+                  },
+                },
+                {
+                  title: 'Save Dex certificate',
+                  task: async (ctx: any, task: any) => {
+                    const dexCaCertificateFilePath = this.getDexCaCertificateFilePath()
+                    fs.writeFileSync(dexCaCertificateFilePath, ctx[DexContextKeys.DEX_CA_CRT])
+                    task.title = `${task.title}...[OK: ${dexCaCertificateFilePath}]`
+                  },
+                },
+                {
+                  title: 'Add Dex certificate to Eclipse Che certificates bundle',
+                  task: async (ctx: any, task: any) => {
+                    if (await this.kube.isConfigMapExists(DexCaConfigMap.NAME, this.flags.chenamespace)) {
+                      task.title = `${task.title}...[Exists]`
+                    } else {
+                      const dexCa = new V1ConfigMap()
+                      dexCa.metadata = new V1ObjectMeta()
+                      dexCa.metadata.name = DexCaConfigMap.NAME
+                      dexCa.metadata.labels = DexCaConfigMap.LABELS
+                      dexCa.data = { 'ca.crt': ctx[DexContextKeys.DEX_CA_CRT] }
+
+                      await this.kube.createNamespacedConfigMap(this.flags.chenamespace, dexCa)
```

Comment:
Where is the labels for marking a config map as a trusted CAs bundle config map added?

Reply:
Sorry, missed it. Thanks for pointing out.